cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -98,10 +98,13 @@ def finish_flow_cell_temp(self, flow_cell_directory_name: str) -> None:
             flow_cell_directory=flow_cell_directory_path
         )
 
-        parsed_flow_cell: FlowCellDirectoryData = self.parse_flow_cell_directory_data(
-            flow_cell_directory=flow_cell_directory_path,
-            bcl_converter=bcl_converter,
-        )
+        try:
+            parsed_flow_cell: FlowCellDirectoryData = self.parse_flow_cell_directory_data(
+                flow_cell_directory=flow_cell_directory_path,
+                bcl_converter=bcl_converter,
+            )
+        except FlowCellError:
+            return
```

Comment:
I always like to print a warning in these cases including the name of the failed flow cell